src/textwin.cpp

```diff
@@ -929,7 +991,7 @@ void TextWindow::Paint() {
 
     canvas->SetLighting(lighting);
     canvas->SetCamera(camera);
-    canvas->StartFrame();
+    canvas->NewFrame();
```

Comment:
I don't understand what this change has to do with the others - is it perhaps an accident? I've now got:

```
../src/textwin.cpp:994:13: error: no member named 'NewFrame' in 'SolveSpace::ViewportCanvas'
    canvas->NewFrame();
    ~~~~~~  ^
```

in clang 10 (after merging from master)